api.js

```diff
@@ -327,14 +327,19 @@ Api.prototype.syncPeers = function (req, res, m, q) {
   }
 }
 
+Api.prototype.syncClear = function (req, res) {
+  this.core.sync.clear()
```

Comment:
Should this be [`clearState`](https://github.com/digidem/mapeo-core/blob/a77e3b33b462cae92de6d4c1323fcb6b5600fdb8/sync.js#L56)?

Reply:
yeah